db/migrate/20220716113150_create_comments.rb

```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Nice job creating tables however this table should be associated with the user and post tables. Kindly use a migration like this :point_down

```.rb
class CreateComments < ActiveRecord::Migration[7.0]
  def change
    create_table :comments do |t|
      t.string :text
      t.references :post, foreign_key: true
      t.references :author, foreign_key: { to_table: 'users' }

      t.timestamps
    end
  end
end
```

Reply:
This issue is well resolved! ✔️